x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/indexlifecycle/SetSingleNodeAllocateStepTests.java

```diff
@@ -99,21 +99,21 @@ public void testPerformActionNoAttrs() throws IOException {
         IndexMetaData indexMetaData = IndexMetaData.builder(randomAlphaOfLength(10)).settings(settings(Version.CURRENT))
                 .numberOfShards(randomIntBetween(1, 5)).numberOfReplicas(randomIntBetween(0, 5)).build();
         Index index = indexMetaData.getIndex();
-        Set<String> validNodeNames = new HashSet<>();
+        Set<String> validNodeIds = new HashSet<>();
         Settings validNodeSettings = Settings.EMPTY;
         DiscoveryNodes.Builder nodes = DiscoveryNodes.builder();
         int numNodes = randomIntBetween(1, 20);
         for (int i = 0; i < numNodes; i++) {
             String nodeId = "node_id_" + i;
             String nodeName = "node_" + i;
             int nodePort = 9300 + i;
-            Settings nodeSettings = Settings.builder().put(validNodeSettings).put("node.name", nodeName).build();
+            Settings nodeSettings = Settings.builder().put(validNodeSettings).put(Node.NODE_NAME_SETTING.getKey(), nodeName).build();
```

Comment:
Do we still need to set the node names here if we're not doing anything with them?

Reply:
no, I felt it was arbitrary since the whole goal is to "fake a node" in unit tests anyways, so I decided to leave it unchanged.